content/blog/2022-06-14-DVC-VS-Code-extension.md

```diff
@@ -0,0 +1,146 @@
+your machine learning projects. You can control datasets, run experiments, view
+metrics, create plots, and much more. All in one place in your IDE.
+
+[Install the DVC extension from the marketplace to get
+started.](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
+
+# Why an extension?
+
+We built DVC to expand upon the Git workflow. This approach has made it a
+powerful tool that can easily be picked up by users familiar with Git. However,
+it can be more difficult for those less familiar with CLI tools to get
+accustomed to using DVC. Moreover, while DVC provides many useful
+visualizations, a CLI tool can only take things so far.
+
+With this extension, we want to achieve the following:
+
+- Move the entire data science workflow into your IDE
+- Enhance existing visualizations and provide new ones
+- Make DVC more accessible by hiding the complexity of the CLI
+
+As data scientists, DVC is our toolbox. This extension turns VS Code into our
+workshop.
+
+# Why Visual Studio Code
+
+One of [our core beliefs at Iterative](https://iterative.ai/why-iterative/) is
+that our products should couple tightly with existing tools and processes for
+data science projects. [VS Code is the IDE of choice for
+many](https://insights.stackoverflow.com/survey/2021#section-most-popular-technologies-integrated-development-environment),
+including many of us at Iterative. Here are a few reasons why we really like it:
+
+VS Code...
+
+- is open source and lightweight
+- has excellent support for Git workflows
+- supports Jupyter Notebooks
+- comes with remote development and execution built-in
+- has a rich ecosystem of extensions
+
+# Features
+
+Our first course of action was to include all DVC features related to data
+management and experiments in the extension. Here are the main features that are
+available when you install the extension right now:
+
+## DVC View: experiments, metrics, and plots
+
+An entirely new view to manage your experiments. Run new experiments, analyze
+their outcomes through plots and metrics, and compare them to find the best
+model.
+
+![Experiment tables to compare different
+experiments](/uploads/images/2022-06-14/dvc-view-experiments-table.png)
+
+![Plots to visualize your
+experiments](/uploads/images/2022-06-14/dvc-view-plots.png)
+
+## Command palette
+
+The DVC extension makes extensive use of the Command Palette, reducing the need
+to learn commands by heart:
+
+![Use the command palette to quickly find DVC
+features](/uploads/images/2022-06-14/command-palette.gif)
+
+## Source control management
+
+Manage changes to your datasets and models right from the source control panel,
+just like you would with changes to your code. Use `checkout`, `commit`, `add`,
+`push`, and `pull` straight from the interface:
+
+![Get a quick overview of changes to your data in the source control
+panel](/uploads/images/2022-06-14/source-control.png)
```

Comment:
This pic would look better if it were wider. It's kind of small/not legible, especially compared to the rest.